deviations/viewbase.py

```diff
@@ -296,4 +296,4 @@ def get_submitters(form_data: Dict[str, Any]) -> models.QuerySet[UserProfile]:
     return UserProfile.objects.filter(
         models.Q(id__in=form_data.get('submitter', []))
         | models.Q(taggings__tag__in=form_data.get('submitter_tag', []))
-    )
+    ).distinct('id')
```

Comment:
The Django documentation very clearly says that the `fields` argument is only supported on PostgreSQL. They also say that `order_by()` is required. But doesn't this work without the `'id'` argument too? We really want only unique UserProfiles and distinct does that even if it compares all fields (the normal behaviour), right?

https://docs.djangoproject.com/en/3.2/ref/models/querysets/#distinct

```suggestion
    ).distinct()
```